File: src/WebClient/mssql.client/src/app/procedure/model/StoredProcedureCreateScript.ts
// Represents a script to create a stored procedure.
export class StoredProcedureCreateScript {
    storedProcedureName: string;
    procedureDefinition: string;
    constructor(init?: Partial<StoredProcedureCreateScript>) {
        Object.assign(this, init);
    }
}

File: src/Services/API/Core/Domain/Database/e/source/repos/MSSQL/src/Services/API/Domain/StoredProcedure/StoredProcedureCreateScript.cs
namespace API.Core.Domain.Database.StoredProcedure
{
    /// <summary>
    /// Represents a script to create a stored procedure.
    /// </summary>
    public class StoredProcedureCreateScript
    {
        /// <summary>
        /// Gets or sets the name of the stored procedure.
        /// </summary>
        public string StoredProcedureName { get; set; }

        /// <summary>
        /// Gets or sets the definition of the procedure.
        /// </summary>
        public string ProcedureDefinition { get; set; }
    }

}

